recipes/mkdocs-click/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "mkdocs-click" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

PyPI's handling of underscores and dashes can be unintuitive:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mkdocs_click-{{ version }}.tar.gz
```